marlowe-playground-client/static/css/panels.css

```diff
@@ -365,11 +373,35 @@ button.minus-btn:hover {
 
 .error-headers {
   font-weight: bold;
-  padding: 0.5em 0em;
+  padding: 1rem 0rem 0.5rem 0rem;
 }
 
 .demo-list li a.disabled {
   color: lightgray;
   pointer-events: none;
   cursor: default;
+}
+
+.rotate-left {
+  rotate: 270;
```

Comment:
What am I missing - Chrome confirms that `rotate` isn't a CSS property.

Reply:
huh? it works in my chrome, does it not work for you?

Reply:
I will check cross browser

Reply:
It seems you use: `transform: rotate(180deg);` up there. `rotate` directly exists but it is not very well supported apparently https://caniuse.com/#search=rotate